src/main/kotlin/com/redhat/devtools/intellij/kubernetes/model/resource/kubernetes/KubernetesReplicas.kt

```diff
@@ -0,0 +1,131 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package com.redhat.devtools.intellij.kubernetes.model.resource.kubernetes
+
+import com.redhat.devtools.intellij.kubernetes.model.context.IActiveContext.ResourcesIn
+import com.redhat.devtools.intellij.kubernetes.model.resource.NonCachingSingleResourceOperator
+import com.redhat.devtools.intellij.kubernetes.model.resource.ResourceKind
+import com.redhat.devtools.intellij.kubernetes.model.resource.openshift.ReplicationControllersOperator
+import io.fabric8.kubernetes.api.model.HasMetadata
+import io.fabric8.kubernetes.api.model.Pod
+import io.fabric8.kubernetes.api.model.ReplicationController
+import io.fabric8.kubernetes.api.model.apps.Deployment
+import io.fabric8.kubernetes.api.model.apps.ReplicaSet
+import io.fabric8.kubernetes.api.model.apps.StatefulSet
+
+open class KubernetesReplicas(
+    protected val operator: NonCachingSingleResourceOperator,
```

Comment:
`operator` appears to be a reserved word in Kotlin. It doesn't appear to be causing any issues in this case, so I don't think changing it is necessary.

Reply:
corrected